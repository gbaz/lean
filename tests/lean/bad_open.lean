open "nat"

open [class] "nat"
