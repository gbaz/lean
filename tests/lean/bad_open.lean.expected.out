bad_open.lean:1:5: error: invalid 'open/export' command, identifier expected
bad_open.lean:3:13: error: invalid 'open/export' command, identifier expected
